test/documentation-content/test-toolkit/1.0/modules/ROOT/pages/process-testing-overview.adoc

```diff
@@ -0,0 +1,23 @@
+= Bonita test toolkit
+
+:description: integration test bonita processes applications junit ci continuous integration
+
```

Comment:
I suggest we only put fake content here (as in other components) for simplicity.